Makefile

```diff
@@ -2,6 +2,19 @@
 build :
 	dune build
 
+.PHONY : npm
```

Comment:
`make npm` seems a bit ambigious to me. Again maybe this is some sort of convention but I’d call it `npm-publish`.

Reply:
I don't think this should be called `npm-publish`, because, as I understand, it just packages things in a local `_release` directory. But it shouldn't be called `npm` either.

Reply:
`make npm-release` ?

Reply:
`-release` has the danger of being interpreted as the verb, which makes it seem dangerous. How about `npm-package`?